azurerm/internal/services/datafactory/data_factory_linked_service_azure_blob_storage_resource.go

```diff
@@ -71,6 +82,38 @@ func resourceDataFactoryLinkedServiceAzureBlobStorage() *schema.Resource {
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
 
+			"use_managed_identity": {
+				Type:          schema.TypeBool,
+				Optional:      true,
+				Default:       false,
+				ConflictsWith: []string{"service_principal_key", "service_principal_id"},
+				AtLeastOneOf:  []string{"service_principal_key", "service_principal_id", "use_managed_identity", "connection_string", "sas_uri"},
```

Comment:
This I think can be simplified to:
```suggestion
				ExactlyOneOf:  []string{"service_principal_id", "use_managed_identity"},
```